.github/workflows/gerrit-compose-required-rtdv3-verify.yaml

```diff
@@ -83,6 +83,8 @@ jobs:
         id: setup-python
         with:
           python-version: "3.8"
+      - name: Install graphviz
+        uses: tlylt/install-graphviz/action.yml@v1.0.0
```

Comment:
There's no reason to specify action.yml here. In fact, it may not work as intended...

Reply:
I had done that due to actionlint requiring the path. Unfortunately, I didn't understand the implication of referencing an action directly, rather than a workflow.